### DISCUSSION TITLE
Are the .sig files in C:\Program Files\Sandboxie-Plus used and why don't have all .exe corresponding .sig files?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#2127] [Q&A Feedback] [answered]
C:\Program Files\Sandboxie-Plus contains .sig files for some .exe files, like SandMan.exe has SandMan.exe.sig. Are those used by anything to validate the files before they are executed?

And why don't have all .exe files corresponding .sig files?

**Comment by DavidXanatos** [comment, marked as answer]
The files are indeed used to validate the exe's by the driver, files
without a proper signature will not get to use the full api.
Files without a sig file don't need to use this api.

Am Sa., 13. Aug. 2022 um 11:42 Uhr schrieb Sebastian G. <
***@***.***>:

**Reply by bastik-1001** [reply]
Thank you.
